src/components/ContribGraph.js

```diff
@@ -0,0 +1,14 @@
+import React, { useEffect } from "react";
+import GitHubCalendar from "github-calendar";
+import "github-calendar/dist/github-calendar.min.js";
+import "github-calendar/dist/github-calendar-responsive.css";
+import "./ContribGraph.css";
```

Comment:
nici de acesta

Reply:
pai cum o sa se afiseze calendarul sara sa fac import import GitHubCalendar from "github-calendar";

Reply:
deci atunci cand tu importi `github-calendar/dist/github-calendar.min.js` GithubCalendar devine o variabila globala

Reply:
Bine, dar imi zice ca nu e definit 
`Failed to compile.

src\components\ContribGraph.js
  Line 7:5:  'GitHubCalendar' is not defined  no-undef 

Search for the keywords to learn more about each error.`

Reply:
``Failed to compile.

src\components\ContribGraph.js
  Line 7:5:  'GitHubCalendar' is not defined  no-undef 

Search for the keywords to learn more about each error.``

Reply:
window.GithubCalendar

Reply:
De ce window.GithubCalendar??

Reply:
El se pune ca o variabila globala, astfel ca sa o poti accesa de oriunde

Reply:
bine am schimbat, dar acum ce nu-i place ?
![image](https://user-images.githubusercontent.com/51832550/107263053-f45fad80-6a49-11eb-9c14-1043588f7fae.png)

Reply:
`useEffect(() => {
    window.GitHubCalendar(".calendar", user, { responsive: true });
  });
`

Reply:
okay atunci hai sa incercam importul sa-l facem un pic altfel

Reply:
pot sa te sun pe teams?

Reply:
da sigur